index.html

```diff
@@ -39,8 +40,143 @@ <h1 class="headline-title">I'm Aniekan <br> Glad to see <br> you!</h1>
+                    <div class="item3">
+                        <ul>
+                            <li><a href="#" class="canopy">Canopy</a></li>
+                            <li><a href="#"><img src="image/Counter.png" alt="dot"></a></li>
+                            <li><a href="#">Back End Dev</a></li>
+                            <li><a href="#"><img src="image/Counter.png" alt="dot"></a></li>
+                            <li><a href="#">2015</a></li>
+                        </ul>
+                    </div>
+                        <div class="item4"><p>A daily selection of privately personalized reads; no accounts, or sign-ups required</p></div>
+                        <div class="item5">
+                            <ul class="inner-item">
+                                <li class="skill"><a href="#">html</a></li>
+                                <li class="skill"><a href="#">css</a></li>
+                                <li class="skill"><a href="#">javaScript</a></li>
+                            </ul>
+                        </div>
+                        <div class="item6"><a href="#">See Project</a></div>
+                </div>
+                <div class="property">
+                    <div class="item1"><img src="image/Snapshoot-Portfolio4.png" alt="Tonic"></div>
+                    <div class="item2"><h2>Tonic</h2></div>
+                    <div class="item3">
+                        <ul>
+                            <li><a href="#" class="canopy">Canopy</a></li>
+                            <li><a href="#"><img src="image/Counter.png" alt="dot"></a></li>
+                            <li><a href="#">Back End Dev</a></li>
+                            <li><a href="#"><img src="image/Counter.png" alt="dot"></a></li>
+                            <li><a href="#">2015</a></li>
+                        </ul>
+                    </div>
+                        <div class="item4"><p>A daily selection of privately personalized reads; no accounts, or sign-ups required</p></div>
+                        <div class="item5">
+                            <ul class="inner-item">
+                                <li class="skill"><a href="#">html</a></li>
+                                <li class="skill"><a href="#">css</a></li>
+                                <li class="skill"><a href="#">javaScript</a></li>
+                            </ul>
+                        </div>
+                        <div class="item6"><a href="#">See Project</a></div>
+                </div>
+            </div>
+            <section class="about-me">
+                <h2 class="headline">About <br> Myself</h2>
+               
+                <p class="about-me-text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+           
+                <p class="text">LET'S CONNECT</p>
```

Comment:
- [ ] [OPTIONAL] as per the requirements of the project. You can make this a heading tag any of the `h1`..`h6` tags would be fine.
something like this is great 👇:
```suggestion
                <h3 class="text">LET'S CONNECT</h3>
```